src/components/Tabs/ScrollableTabs/ScrollableTabs.tsx

```diff
@@ -0,0 +1,175 @@
+import * as React from 'react';
+import { Tab, TabItem } from '../Tab/Tab';
+import { ALIGNMENT as Alignment, VARIANT as Variant } from '../../Tabs';
+import { animateElementByProp } from '../../../common/animations';
+import style from './ScrollableTabs.st.css';
+
+const isEqual = require('lodash/isEqual');
```

Comment:
any special reason for `require` over `import` syntax?

Reply:
typescript issues. I'll add a @ts-ignore comment